source/api/extension/text-granularity/index.md

```diff
@@ -0,0 +1,158 @@
+  "type": "Annotation",
+  "textGranularity": "line",
+  "motivation": ["supplementing"],
+  "body": {
+    "type": "TextualBody",
+    "language": "la",
+    "value": "arma virumque cano, Troiae qui primus ab oris"
+   },
+   "target": {
+     "type": "SpecificResource",
+     "source": "https://example.org/aeneid/canvas/1r",
+     "selector": {
+        "type": "FragmentSelector",
+        "value": "xywh=500,1100,3500,100"
+     }
+   }
+```
+Alternatively, the body might be an [external web resource](https://www.w3.org/TR/annotation-model/#external-web-resources).  For example, the Body could use an XPath selector to identify the transcription of a paragraph of text within an XML document.
+
+```json-doc
+{
+  "id": "https://example.org/iiif/aeneid/book1/transcription-line2",
+  "type": "Annotation",
+  "textGranularity": "line",
+  "motivation": ["supplementing"],
+  "body": {
+    "type": "SpecificResource",
+    "source": "http://example.org/aeneid-tei.xml",
+    "selector": {
+      "type": "XPathSelector",
+      "value": "/TEI.2/text/body/div1/l[2]"
+    }
+   },
+   "target": {
+     "type": "SpecificResource",
+     "source": "https://example.org/aeneid/canvas/1r",
+     "selector": {
+        "type": "FragmentSelector",
+        "value": "xywh=500,1100,3500,100"
+     }
+   }
+}
+```
+
+An exact alignment in the level of granularity between the Body and Target may be impractical. For example, a single manuscript paragraph in the targeted Canvas may be divided across multiple paragraphs in a transcription derived from a printed version of the work, or a single character may be transcribed by multiple characters in an alternative script.
+
+The `motivation` value of the associated Annotation _MUST_ have the value `supplementing` when the content of the annotation is derived from the Canvas.  This is a requirement of the [Presentation API][prezi30-canvas].
+
+## 4. Linked Data Context
+
+The URI of the JSON-LD context for this extension is `http://iiif.io/api/extension/text-granularity/context.json`.  Consult the [Linked Data Context and Extensions][prezi30-ldce] section of the Presentation API for further guidance on its use within the `@context` property.
+
+```json-doc
+{
+  "@context": [
+     "http://iiif.io/api/extension/text-granularity/context.json",
+     "http://iiif.io/api/presentation/3/context.json"
+  ]
+}
+```
+
+## 5. Implementation Note
+
+The Text Granularity Levels defined above were derived from a survey of commonly used formats for OCR output such as hOCR, ALTO, and ABBYY, as well the output produced by the Google Cloud Vision API. The findings are available in the [Granularities in Different OCR Formats document](https://docs.google.com/document/d/13R7Dk-AA-ALZ5i3fzAS3g66umWjiYaD7NVr8QiYb19E/edit#heading=h.q6phhnqlv5sf). The following table maps Text Granularity Levels to structures found in these output formats and is intended as an aid to implementers.
```

Comment:
Should the Google doc be translated to Markdown included on the website?

Reply:
I'm going to remove this, after reflection I don't think the document adds much.